demo/rails/app/views/captcha/index.html.erb

```diff
@@ -36,11 +36,21 @@
     <%= submit_tag %>
   <% end %>
   <script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" async defer></script>
-  <%= link_to 'Single ?', '?' %>
+<% elsif params[:invisible] %>
+  <script>
+    function onSubmitCallback() {
```

Comment:
can't have a global method with such a generic name ... also should not use inline js to be CSP compliant